diagnostic.md

```diff
@@ -6,70 +6,70 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+@books = Book.find(params[4])
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+In the routes.rb file
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+This command runs migrations that have not been run yet, specifically for new databases or controllers.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+Scaffolding things creates a resource called things where rails g controller things creates just a controller for things.
```

Comment:
The first command generates routes, a controller and model for foo. The second generates just a foo controller.